playbooks/update.yml

```diff
@@ -19,6 +28,11 @@
   hosts: "{{ cifmw_target_host | default('localhost') }}"
   gather_facts: false
   tasks:
+    - name: Set update step to Update Repo and OpenStack Services Containers
+      ansible.builtin.command:
+        cmd: >
+          {{ cifmw_basedir }}/tests/update/update_step.sh
+          Update Repo and OpenStack Services Containers
```

Comment:
nit: maybe in the single quote will be better?

Reply:
This avoid long line, would rather keep it as it is.